tutorials/tutorial11_traffic_lights.ipynb

```diff
@@ -355,7 +383,7 @@
     "self.last_change = np.zeros((self.rows * self.cols, 1))\n",
     "# keeps track of the direction of the intersection (the direction that is currently being allowed to flow. 0 indicates flow from top to bottom, and 1 indicates flow from left to right.)\n",
     "self.direction = np.zeros((self.rows * self.cols, 1))\n",
-    "# value of 0 indicates that the intersection is in a red-yellow state. 1 indicates that the intersection is in a red-green state.\n",
+    "# value of 1 indicates that the intersection is in a red-yellow state. 0 indicates that the intersection is in a red-green state.\n",
```

Comment:
nit: it's not clear what a red-yellow state is. You mean rYrY right? I think this is held over text so it's not your mistake but I think it's sort of unclear what it means.